src/temp/Project/Filter.tsx

```diff
@@ -0,0 +1,48 @@
+import FilterIcon from '../icons/FilterIcon';
+import { Button } from '@mui/material';
+import style from './Filter.module.scss';
+import { useTranslation } from 'next-i18next';
+
+interface FilterProps {
+  activeFilter: boolean;
+  projectList: string[];
```

Comment:
This isn't a list of projects, so why call this `projectList`? Perhaps it should be `projectTypes` or `ecosystemTypes` (if it contains only a list of ecosystems).